suite-native/device-manager/src/components/DeviceManagerModal.tsx

```diff
@@ -1,47 +1,62 @@
 import { GestureResponderEvent, Modal, Pressable } from 'react-native';
 import { ReactNode } from 'react';
 import { useSafeAreaInsets, EdgeInsets } from 'react-native-safe-area-context';
-import Animated, { SlideInUp } from 'react-native-reanimated';
+import Animated, { FadeIn, SlideInUp } from 'react-native-reanimated';
+import { useSelector } from 'react-redux';
 
-import { ScreenHeaderWrapper, VStack } from '@suite-native/atoms';
+import { ScreenHeaderWrapper, Box, HStack } from '@suite-native/atoms';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { selectDevice } from '@suite-common/wallet-core';
 
-import { DeviceSwitch } from './DeviceSwitch';
 import { useDeviceManager } from '../hooks/useDeviceManager';
+import { DeviceItemContent } from './DeviceItem/DeviceItemContent';
 
 type DeviceManagerModalProps = {
     children: ReactNode;
+    customSwitchRightView?: ReactNode;
+    onClose?: () => void;
```

Comment:
Should be required

Reply:
it feels better to handle optionality here than having to write 
`onClose={() => {}}>`
in `PortfolioTrackerDeviceManagerContent`